app/javascript/components/pages/GooseyIndex.js

```diff
@@ -1,38 +1,53 @@
 import React, { Component } from "react";
-import { Card, CardTitle, Col, Row } from "reactstrap";
+import { Card, CardTitle, Col, ListGroupItemHeading, Row } from "reactstrap";
 import { NavLink } from "react-router-dom";
 
 export default class GooseyIndex extends Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      listings: [],
+    };
+  }
 
+  componentDidMount() {
+    this.readListing();
+  }
 
-
+  readListing = () => {
+    fetch("/listings")
+      .then((response) => {
+        return response.json();
+      })
+      .then((payload) => {
+        this.setState({ listings: payload });
+      })
+      .catch((errors) => {
+        console.log("Gooseys read errors:", errors);
+      });
+  };
 
   render() {
     return (
       <>
-        <h3>All the Listings</h3>
-        <Row className="cards">
-          {this.props.listings.map((listing) => {
-            return (
-              <Col sm="4" key={listing.id}>
-                <Card body>
-                  <CardTitle>
-                    <h5>{listing.location_name}</h5>
-                    <h5>
-                      {listing.specials}, {listing.happy_hours}{" "}
-                      {listing.location_category}, {listing.location_image},
-                    </h5>
-                    <p className="card-button">
-                      <NavLink to={`/listing/${listing.id}`}></NavLink>
-                    </p>
-                  </CardTitle>
-                </Card>
-              </Col>
-            );
-          })}
-        </Row>
-      </>
-    );
-  }
+  <div className="cards">
+    {this.state.listings.map((listing) => {
+      return (
+        <div className="card" key ={listing.id}>
+          <h2 className="card-header">{listing.location_name}</h2>
+          <img className="card-img-top" src={listing.location_image}/> 
+          <div className="card-body">
+            <p className="card-text">{listing.specials}</p>
+            <p className="card-text">{listing.location_category}</p>
+            <p className="card-text">{listing.happy_hours}</p>
+          </div>
+        </div>
+      );
+    })}
+  </div>
+
+</>
+);
```

Comment:
This section should be tabbed over to be nested inside the react fragment on line 33.  The return closing parenthesis should be aligned up under the return.